### ISSUE TITLE
add msquic to Centos 7 Helix queue

### ISSUE THREAD

**Issue by wfurt** [dotnet/arcade#12540]
This is follow-up on https://github.com/dotnet/arcade/issues/10253#issuecomment-1215320634
```
// from https://engdata.kusto.windows.net
WorkItems
| where JobName in (
"6a1d3b91-f1ae-40a3-8725-0bb3f45d793b", // Centos 7 doesn't work
"fb4186a2-dded-4c9a-8ba7-20314b96acb0", // RHEL7 good
"fe8209d6-d1a3-4a47-b534-2a057ac3aae9", // SLES 12 good
"81f9f8ee-4058-453e-8f82-8d061f46521b", // SLES 15 good
"d00bf0eb-14d0-4f47-b59f-c5048a5f144d", // Ubuntu 16.04 doesn't work
"adcf6c0b-1044-4dde-ada2-5c6137d01b0f", // Ubuntu 18.04 good
"469631c7-3f8a-46f5-ae21-5bf43e44b060", // Ubuntu 20.04 is good
"963594b0-7a9a-4d04-97cc-6735a67e8e10") // Ubuntu 22.04 is good
| where Name != JobName
```

I don't know if it is just time lapse, but it should be possible now. see https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/800 for detail. 

openssl11 needed my msquic comes from EPEL repo. So it needs to be added if it is not already there.

cc: @manickap @CarnaViire @MattGal

**Comment by MattGal** [comment]
@wfurt given you have the linked PR open, is this a request to add it to our non-docker images, or just a tracking issue about doing the work to support it in docker?

**Comment by wfurt** [comment]
We also have test runs in full Helix queue and it is part of the hunt for complete coverage. 
So yes, this is request to add it to non-docker images unless we decided to abandon it in runtime ci (and for example jump  to 9)
In practice, this may be blocked on https://github.com/microsoft/msquic/issues/3422.

**Comment by MattGal** [comment]
OK, tossing on FROps backlog.

**Comment by MattGal** [comment]
With https://github.com/microsoft/msquic/issues/3422 still open and the suggested fix not merged, I am checking with @wfurt about this this morning.

**Comment by MattGal** [comment]
Chatted with @wfurt  today.  

The work in the helix machines repo is already done.  That is, we already install libmsquick from the Microsoft package source on all Centos 7 images, and routinely update the base images from Azure.  

Every time this happens whatever is on the Microsoft package feed source is fetched and installed, so whenever this is updated our existing processes will address the issue.  As we don't want to introduce building from source into the plain helix images, there is no work for DncEng to do here, so I am closing the issue.

**Comment by wfurt** [comment]
I check the Helix repro machine and looks good.
```
[toweinfu@toweinfu-c7 ~]$ rpm -qa| grep quic
libmsquic-2.1.7-1.x86_64
```

so it is on auto-update and it will pick up fixes when ready. It was not clear to me from the original description. 
I think we will need to wait for MsQuic to roll-out next release with fix.

**Comment by wfurt** [comment]
We generally need to do
```
$ sudo yum install epel-release
$ sudo yum install openssl11-libs
```

**Comment by MattGal** [comment]
Cheap to add, reopening and I'll add that.

**Comment by wfurt** [comment]
Thanks @MattGal. We will _also_ need fix from MsQuic but since the Queue is on regular updates we will just need to wait.

**Comment by MattGal** [comment]
should be handled by https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/29760, rolling out this week

**Comment by MattGal** [comment]
This rolled out today.  If you continue to have issues with Centos.7 images please ping dnceng.

**Comment by wfurt** [comment]
thanks @MattGal  for quick fix
